site/config.toml

```diff
@@ -49,6 +49,12 @@ disqusShortname = "checkly"
 	pre="/docs/images/icons/alerting.svg"
```

Comment:
General comment. Does it make sense to split all the resources into single articles as you did, or should we have one long doc?